diffrax/brownian/base.py

```diff
@@ -1,14 +1,49 @@
 import abc
+from typing import Optional, Union
 
-from ..custom_types import Array, PyTree, Scalar
+import equinox as eqx
+import jax
+from jax import tree_util as jtu
+
+from ..custom_types import Array, LevyVal, PyTree, Scalar
 from ..path import AbstractPath
 
 
+def _levy_tree_transpose(tree_shape, spacetime_levyarea, tree):
```

Comment:
Remove the leading underscore, as this is imported into other files, so it isn't private.

Reply:
Sounds good! I also moved it to custom_types right under the definition of LevyVal, since any changes to LevyVal (e.g. adding space-time-time Levy area) will have to be reflected in levy_tree_transpose, so it's nice to have them close.